test/e2e/main/main_test.go

```diff
@@ -629,7 +637,7 @@ func TestEndpointEncryptionUpdate(t *testing.T) {
 // Test if single node working correctly
 func genericTestForContainerUpdated(ispn ispnv1.Infinispan, modifier func(*ispnv1.Infinispan), verifier func(*ispnv1.Infinispan, *appsv1.StatefulSet)) {
 	testKube.CreateInfinispan(&ispn, tutils.Namespace)
-	defer testKube.DeleteInfinispan(&ispn, tutils.SinglePodTimeout)
+	defer testKube.CleanNamespaceAndLogOnPanic(tutils.Namespace, ispn.Labels)
```

Comment:
@durgeshanaokar `genericTestForContainerUpdated` calls from other tests where Infinispan CR's are not labeled. Please review all tests. That's the reason why `TestEndpointAuthenticationUpdate` is failed.

Reply:
@dmvolod thank for the update, I think we also need to make changes into backup_restore_test.go

Reply:
This is not required, backup/restore never runs in parallel and we need to clean whole namespace for it. Please revert changes to backup/restore